docs/source/workflows/whisper-transcripts-cleanup.rst

```diff
@@ -0,0 +1,42 @@
+==========================
+Cleaning a Whisper Transcript
+==========================
+
+Whisper AI will generate transcripts with a variety of flaws and issues. However, to save time, prioritize fixing certain issues over others.
+
+---------------------
+Goals for Cleanup
+---------------------
+
+* Whisper transcripts should be as accurate to the audio as possible. No additions or omissions.
+* Whisper transcripts should be readable. Grammar should be accurate enough so that it is understandable without audio and can be read.
+* Whisper transcripts should be searchable. It is expected that users will search for specific keywords such as the name of a person or the title of a work. Therefore, these names should be spelled correctly.
+
+---------------------
+How to clean a transcript
+---------------------
+
+* Remove all hallucinations. Hallucinations can be identified by blocks of red text.
+* Fill in omissions. This can be done either by listening to the audio and transcribing by hand or by splicing together multiple AI-generated transcripts. If you do the latter, document where you added text and what file you pulled your addition from.
+* Capitalize proper names and beginnings of sentences.
+* Add puntuation as needed, especially periods and punctuation used within the title of a work.
+* Make sure the following are spelled correctly: Names of individuals, places, organizations, events, groups, works. This includes fictional varieties of each (ie characters, fictional organizations, imaginary places).
```

Comment:
Eventually, we should make this more contextual depending on the content of the video.